pywincffi/wintypes/__init__.py

```diff
@@ -0,0 +1,113 @@
+"""
+Windows Types Package
+====================
+
+Provides user accessible types corresponding to the respective Windows types
+used across the exposed APIs.
+"""
+
+
+from six import integer_types
+
+from pywincffi.core import typesbase
+from pywincffi.core import dist
+
+
+_FFI, _ = dist.load()
```

Comment:
So this is why the doc build step fails.  

When you call dist.load() it will either import `_pywincffi` or try to build it if it does not exist.  Travis builds the documentation on Linux just like [readthedocs.org](https://readthedocs.org/projects/pywincffi/builds/) across multiple Python versions so if there's a problem like this it's easy to discover while doing development versus after the fact.  Of course Linux does not have access the headers and libraries that Windows does so this will fail.

I suggest using `dist.load()` inside of each of the functions in this module like we do elsewhere in pywincffi.